### DISCUSSION TITLE
NSSecureCoding support

### DISCUSSION THREAD

**Discussion by Juan2662** [firebase/firebase-ios-sdk#12061] [Pitches]
Hi guys, I notice that some chunks in the code in iOS (like FirebaseMessaging) is using NSCoding protocol
Can we have support for NSSecureCoding?

https://github.com/firebase/firebase-ios-sdk/blob/aec227be2b101577e0b531a08ec057ef01e13703/FirebaseMessaging/Sources/Token/FIRMessagingTokenInfo.h#L28

**Comment by paulb777** [comment]
@Juan2662 Thanks for raising this. We've been incrementally adding `NSSecureCoding` support across Firebase and happy to review PR to help us accelerate.

**Reply by Juan2662** [reply]
Thanks for your answer @paulb777
When can we have full support for NSSecureCoding?

**Reply by paulb777** [reply]
Please open a feature request to help us prioritize: https://github.com/firebase/firebase-ios-sdk/issues/new/choose

**Reply by paulb777** [reply]
Opened at #12075. Let's continue the discussion there.
